### PR TITLE
[Issue#47] Fix alignment of block quote

### PR DESCRIPTION
**Pull request by KyleDolezal**
**Overview**
This PR addresses a concern around the appearance of the block quote by changing the justification of block quotes.

**Problem**
The alignment of the block quote is looking a little wonky on smaller screen sizes.

**Solution**
Change the justification for the block quote from right justified to left justified.

### CODE REVIEW

**Review by amy-corson — CHANGES_REQUESTED** [review]
Hello! We use SCSS compiled in to CSS for this project, so the SCSS file for the header will need to be changed, and you can use an extension such as `Watch Sass` to compile! If you can update the scss, change should be good to go! Let me know if you have questions

**Comment by KyleDolezal** [comment]
> Hello! We use SCSS compiled in to CSS for this project, so the SCSS file for the header will need to be changed, and you can use an extension such as `Watch Sass` to compile! If you can update the scss, change should be good to go! Let me know if you have questions

Thanks for the tip! I tried to make the change via scss. Just let me know if there's any other ideas.

File: src/css/main.css [hunk 1]
```diff
@@ -1048,4 +1048,6 @@ body {
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
-}
\ No newline at end of file
+}
+
+/*# sourceMappingURL=main.css.map */
```
Comment (amy-corson) [inline src/css/main.css:1053]: hmm what's this guy doing down here! Can this be safely removed?

**Review by amy-corson — APPROVED** [review]
Looks good, just one quick question on a comment within the css file. 

And I'm so sorry for the late late late review!

**Comment by hkier** [comment]
It appears this PR is capturing the update to main.css after scss compiles the file.  I think we can ignore the change to the main.css file but I have to double check.

**Comment by hkier** [comment]
I have taken a close look at the PR and discovered it is actually 1 commit ahead and 39 commits behind main.  The easiest way to fix the problem is to close this PR, create a new branch with the change and then merge it into main.

**Comment by hkier** [comment]
out of date

**Pull request closed without merging**